nsf/layerwise_training.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: New approach for DNN architecture design
+permalink: /nsf/layerwise_training.md
```

Comment:
Cannot end in .md. Also, so our addresses are consistent, change this to `/nsf/year3/layerwise_training`